### ISSUE TITLE
After executing set_id, the 'aria-labelledby' value of a created node is wrong.

### ISSUE THREAD

**Issue by AbitSoft** [vakata/jstree#1380]
Hi, I'm using jstree 3.2.1.
I think the 'aria-labelledby' has wrong value under the specific condition.

[Source] ---
[index.txt](https://github.com/vakata/jstree/files/170796/index.txt)

[Steps] ---
1. Click the [+file] button.
2. Confirm the 'aria-labelledby' value of the created node.
3. Click the [+file] button again.
4. Confirm the 'aria-labelledby' values of the created two nodes.

[Result] ---
In the first created node, the 'aria-labelledby' value was 'temp_90001_anchor'. (This value should be 'new_101_anchor'.)

**Comment by vakata** [comment]
it is fixed, but the fix might not be what you expected - the anchor's ID should not have been modified in the first place. The good news is that the link between labelled-by and the ID was never broken, the only issue was that the anchor ID was changed - set ID should not change that.

Best regards,
Ivan

**Comment by AbitSoft** [comment]
Thank you very much for explaining details. I understand about this specification. 

Best regards, 
AbitSoft Keiko
